### ISSUE TITLE
gulp-jsxcs injects too much, ironically causing jscx to fail

### ISSUE THREAD

**Issue by Pomax** [omakasecorp/gulp-jsxcs#9]
running gulp-jsxcs on the following code:

```
var React = require("react");
var Gallery = require("./Gallery.jsx");
var loginServer = "http://localhost:3000";
React.render(<Gallery host={loginServer} />, document.getElementById("app"));

```

yields the following errors:

```
Error: Missing line feed at file end at app.js :
     4 |var loginServer = "http://localhost:3000";
     5 |React.render(React.createElement(Gallery, {host: loginServer}), document.getElementById("app")); // jscs:disable
     6 | // jscs:enable
--------^

Comments must start with an uppercase letter, unless it is part of a textblock at app.js :
     4 |var loginServer = "http://localhost:3000";
     5 |React.render(React.createElement(Gallery, {host: loginServer}), document.getElementById("app")); // jscs:disable
     6 | // jscs:enable
---------^
```

Neither of those things are actually wrong in the original file, but are caused by the comment injections from gulp-jsxcs.

**Comment by agschwender** [comment]
Fixed in f9f56f44bbe453d59fc6a75ebc42390c704133ba or on npm as [0.1.7](https://www.npmjs.com/package/gulp-jsxcs)

**Comment by Pomax** [comment]
nice! thank you very much
